src/cartesian_mesh.cc

```diff
@@ -0,0 +1,8 @@
+#include "flyft/cartesian_mesh.h"
+
+#include <cmath>
+
+double CartesianMesh::volume(int i) const
+    {
+    return step_;
```

Comment:
Perfect! I think we should start adding the cross-sectional area of the `CartesianMesh` in so that this is a true volume. However, I don't want to make too many changes at once so for now...

Can you add a private member variable to `CartesianMesh` called like `area_` or `cross_area_` (whatever you prefer), give it a default value of `1.0` in the constructor, then use it here like:
```suggestion
    return area_*step_;
```